src/backend/api/Fusion.Resources.Api/Authorization/Extensions/RequirementsBuilderExtensions.cs

```diff
@@ -158,52 +158,12 @@ public static IAuthorizationRequirementRule CurrentUserIs(this IAuthorizationReq
             return builder;
         }
 
-        public static IAuthorizationRequirementRule BeContractorInProject(this IAuthorizationRequirementRule builder, ProjectIdentifier project)
-        {
-            builder.AddRule(project, new ContractorInProjectRequirement());
-            return builder;
-        }
-
-        public static IAuthorizationRequirementRule BeContractorInContract(this IAuthorizationRequirementRule builder, Guid orgContractId)
-        {
-            builder.AddRule(new ContractorInContractRequirement(orgContractId));
-            return builder;
-        }
-
-        public static IAuthorizationRequirementRule ProjectAccess(this IAuthorizationRequirementRule builder, ProjectAccess level, ProjectIdentifier project)
```

Comment:
Is this related to contracts? (Should it be removed ?)

Reply:
I did not find any references in the code so I removed it